tiny_dnn/core/kernels/avx_kernel_common.h

```diff
@@ -36,22 +36,46 @@
 #endif
 
 inline __m256 madd256_ps(__m256 a, __m256 b, __m256 c) {
+#ifdef CNN_USE_AVX2
```

Comment:
Putting single `#ifdef CNN_USE_AVX2` around all the code should be more readable